### PR TITLE
Add Man page.

### PR DESCRIPTION
**Pull request by rhatdan**
Modify annotation to io.containers.trace

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Fixes: https://github.com/containers/oci-seccomp-bpf-hook/issues/7

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,11 @@ HOOK_BIN_DIR ?= ${PREFIX}/libexec/oci/hooks.d
 ETCDIR ?= /etc
 HOOK_DIR ?= ${PREFIX}/share/containers/oci/hooks.d/
 
-all:
+.PHONY: docs
+docs:
+	$(MAKE) -C docs
+
+all: docs
 	$(GO_BUILD) -o bin/oci-seccomp-bpf-hook $(PROJECT)
```
Comment (vrothberg) [inline Makefile]: Can we add a new `build` target to avoid recompiling the docs when we want to build?

File: docs/oci-seccomp-bpf-hook.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+OCI hook configuration file for poststop
+
+ * /usr/share/containers/oci/hooks.d/oci-seccomp-bpf-hook-stop.json
+ * /etc/containers/oci/hooks.d/oci-seccomp-bpf-hook-stop.json (Override)
+
+
+## SEE ALSO
+podman(1), oci-hooiks(5), seccomp(2)
```
Comment (vrothberg) [inline docs/oci-seccomp-bpf-hook.md]: I think we need to sort the sections 1, 2, 5 and `s/oci-hooiks/oci-hooks/`

File: docs/oci-seccomp-bpf-hook.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## AUTHORS
+Valentin Rothberg <vrothberg@redhat.com>
+Dan Walsh <dwalsh@redhat.com>
```
Comment (vrothberg) [inline docs/oci-seccomp-bpf-hook.md]: `Divyansh Kamboj <agilephoenix815.dk@gmail.com>`

File: Makefile (hunk 1 above)
Comment (rhatdan) [reply Makefile]: Fixed.  Added binary

File: docs/oci-seccomp-bpf-hook.md (hunk 2 above)
Comment (rhatdan) [reply docs/oci-seccomp-bpf-hook.md]: Fixed

File: docs/oci-seccomp-bpf-hook.md (hunk 3 above)
Comment (rhatdan) [reply docs/oci-seccomp-bpf-hook.md]: I wasn't sure if he wanted his email address out there, but I guess it is now.

Comment (weirdwiz) [reply docs/oci-seccomp-bpf-hook.md]: I would love to change that email to `kambojdivyansh2000@gmail.com` :D

Made a pull request : #14

**Pull request merged**